Mobile Buy SDK Sample Apps/Mobile Buy SDK tvOS Sample/Mobile Buy SDK tvOS Sample/CollectionsViewModel.swift

```diff
@@ -27,29 +27,40 @@
 import Foundation
 import Buy
 
-class Collections {
+class CollectionsViewModel: BaseOperation {
     
-    private var collections: [BUYCollection] = []
-    private var collectionOperation: Operation?
-    
-    weak var collectionView: UICollectionView!
+    fileprivate var collections: [BUYCollection] = []
+
     var dataProvider: DataProvider!
     
-    public init(collectionView: UICollectionView, dataProvider: DataProvider) {
-        self.collectionView = collectionView
+    init(dataProvider: DataProvider) {
         self.dataProvider = dataProvider
-        
-        getCollections()
     }
     
-    func count() -> Int {
+    func getCollections(completion: @escaping () -> Void) {
+        let operation = self.dataProvider.getCollections { (collections) in
+            self.collections = collections
+            completion()
+        }
+        self.setCurrentOperation(operation: operation)
+    }
+}
+
+extension CollectionsViewModel: UICollectionViewDataSource {
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: CollectionViewContainerCell.reuseIdentifier, for: indexPath) as? CollectionViewContainerCell
+        let collection = self.collections[indexPath.section] as BUYCollection
+        let productsViewModel = ProductsViewModel(collection: collection, dataProvider: self.dataProvider)
+        cell?.configure(viewModel: productsViewModel, title: collection.title)
```

Comment:
No need to be defensive here, the cell should be implicitly unwrapped.

`let cell = collectionView.dequeueReusableCell(withReuseIdentifier: CollectionViewContainerCell.reuseIdentifier, for: indexPath) as! CollectionViewContainerCell`